pkg/vpcmodel/drawioOutput.go

```diff
@@ -59,62 +61,86 @@ func (d *DrawioOutputFormatter) createDrawioTree() {
 
 func (d *DrawioOutputFormatter) createNodeSets() {
 	for _, vpcConfig := range d.cConfigs {
-		for _, ns := range vpcConfig.NodeSets {
-			if d.showResource(ns) {
-				d.gen.TreeNode(ns)
+		if !vpcConfig.IsMultipleVPCsConfig {
```

Comment:
Can there actually be `NodeSets` in `DrawioOutputFormatter`?

Reply:
You added a condition `!vpcConfig.IsMultipleVPCsConfig` before addressing `NodeSets.` 
My question: is this condition really required? Are there `NodeSets`  defined in the dummy multiple VPC config `vpcConfig.IsMultipleVPCsConfig`? 

Same question in the comments below. w.r.t. the relevant resource in each comment

Reply:
yes - they are all in there, VPCs, SG, and of course nodes